webapp/static/interop-data.json

```diff
@@ -299,7 +299,8 @@
         "scores_over_time": [
           { "date": "2023-03-08", "score": 600 },
           { "date": "2023-05-13", "score": 700 },
-          { "date": "2023-06-27", "score": 780 }
+          { "date": "2023-06-27", "score": 780 },
+          { "date": "2023-09-05", "score": 860 }
```

Comment:
```suggestion
          { "date": "2023-09-05", "score": 860 },
          { "date": "2023-09-27", "score": 870 }
```

Reply:
One serendipitous benefit to waiting too long to review is that the score has gone up in the meantime!